features/step_definitions/deprecated.rb

```diff
@@ -1,7 +1,3 @@
-Given /^we are using a Rails 4.0 compatible version of cucumber$/ do
-  pending
-end
-
 Given /^I'm creating content which uses deprecated input fields$/ do
```

Comment:
This was only for making it green. I'm working in another branch for killing half of bcms (everything that we don't use)